vividus-plugin-applitools/src/main/resources/spring.xml

```diff
@@ -42,6 +42,11 @@
                 <property name="lastRequestInterceptor"><null/></property>
                 <property name="lastResponseInterceptor"><null/></property>
                 <property name="httpResponseHandlers"><list/></property>
+                <property name="firstRequestInterceptor">
+                    <bean class="org.vividus.http.client.LoggingHttpRequestInterceptor">
+                        <constructor-arg value="org.vividus.visual.eyes.ApplitoolsHttpClient" />
```

Comment:
no need to specify constructor arg type?

Reply:
since we use value the String constructor will be used